source/openapi-admin-v3.yaml

```
@@ -5132,55 +5178,517 @@ components:
         name:
           type: string
           description: The name of the authentication provider. This value must be the same as the value of `type`.
         type:
           "$ref": "#/components/schemas/ProviderType"
         disabled:
           type: boolean
-    FullProvider:
+          description: If `true`, the auth provider is disabled. Users cannot log in through the provider until it is enabled.
+    
+    AuthProvider:
+      anyOf:
+        - $ref: "#/components/schemas/AnonymousAuthProvider"
+        - $ref: "#/components/schemas/EmailPasswordAuthProvider"
+        - $ref: "#/components/schemas/ApiKeyAuthProvider"
+        - $ref: "#/components/schemas/CustomJwtAuthProvider"
+        - $ref: "#/components/schemas/CustomFunctionAuthProvider"
+        - $ref: "#/components/schemas/AppleAuthProvider"
+        - $ref: "#/components/schemas/GoogleAuthProvider"
+        - $ref: "#/components/schemas/FacebookAuthProvider"
+    
+    AnonymousAuthProvider:
+      allOf: 
+        - $ref: "#/components/schemas/BaseAuthProvider"
+        - properties:
+            name:
+              const: anonymous
+              description: The name of the authentication provider. For Anonymous authentication, this value is always `anonymous`.
+            type:
+              const: anonymous
+              description: The name of the authentication provider. For Anonymous authentication, this value is always `anonymous`.
+    
+    EmailConfirmationType:
+      oneOf:
+        - $ref: "#/components/schemas/EmailConfirmationAutomatic"
+        - $ref: "#/components/schemas/EmailConfirmationBuiltIn"
+        - $ref: "#/components/schemas/EmailConfirmationCustomFunction"
+    
+    EmailConfirmationAutomatic:
+      type: object
       properties:
-        _id:
+        autoConfirm:
+          const: true
+          description: If `true`, automatically confirm new user accounts without additional verification.
+    
+    EmailConfirmationBuiltIn:
+      type: object
+      properties:
+        autoConfirm:
+          const: false
+          description: |-
+            If `true`, automatically confirm new user accounts without
+            additional verification. For built-in email account
+            confirmation, set this value to `false`.
+        emailConfirmationUrl:
           type: string
-        name:
+          description: |-
+            The URL included in the email sent to new users as part of
+            the built-in confirmation flow. The email also adds a unique
+            confirmation `token` and `tokenId` as query parameters.
+            
+            This URL should handle user confirmation by passing the
+            confirmation token to a Realm SDK's `confirmUser` function
+            or to the Admin API [Confirm a Pending Email/Password
+            User](https://www.mongodb.com/docs/atlas/app-services/admin/api/v3/#tag/email/operation/adminConfirmPendingUser)
```

```suggestion
            User](#tag/email/operation/adminConfirmPendingUser)
```

this should work b/c same page, but there might be some more redoc-native way to do it.